lib/jalaali/helper.ex

```diff
@@ -0,0 +1,105 @@
+defmodule Jalaali.Helper do
```

Comment:
Can you elaborate what is the idea behind this module? Why would we need these functions exported at all?

Reply:
Some reasons convinced me to create this module. 
* I want to separate some function on main calendar 
* because we do not add all function of new calendar module of new elixir version, Then we can't use calendar module behavior
* I need to create a file to putt all Dialyzer type and call back to clear all project and test
* And we can help the other developer to create own function under our @callback
* I think I can add some helper function to create tools for user in the future, and it should be separate with main file

Reply:
@shahryarjb I thought about this and here is my opinion:

 - Until not needed, Separation of code for future use should not happen.
 - These functions currently in this module are not to be publicly exported as exporting them brings consequences of maintaining them.

If you agree, just get rid of this helper module and then we will merge.

Thanks again for the contribution :heart: